Godeps/_workspace/src/github.com/openshift/source-to-image/pkg/api/types.go

```diff
@@ -91,6 +96,11 @@ type Config struct {
 	// Specify a relative directory inside the application repository that should
 	// be used as a root directory for the application.
 	ContextDir string
+
+	// AllowedUIDs is a list of user ranges of users allowed to run the builder image.
+	// If a range is specified and the builder image uses a non-numeric user or a user
+	// that is outside the specified range, then the build fails.
+	AllowedUIDs user.RangeList
```

Comment:
We could probably separate out some of the logic in the admission controller to an SCC api to help with calls like this.  The relevant components are the total set of SCCs and the `getMatchingSecurityContextConstraints` method in admission which would provide the applicable sets.  That could then be passed to something that would return the blocks of allowable uids supported.